sdks/python/apache_beam/runners/portability/fn_api_runner_test.py

```diff
@@ -215,9 +203,6 @@ def test_gbk_side_input(self):
           main | beam.Map(lambda a, b: (a, b), beam.pvalue.AsDict(side)),
           equal_to([(None, {'a': [1]})]))
 
-  @unittest.skipIf(sys.version_info[0] == 3 and
```

Comment:
Is this test actually passing? I am seeing errors, but maybe it is due to  the fact I am using a different Python 3 version than we have on Jenkins.
```
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner_test.py", line 215, in test_multimap_side_input
    equal_to([('a', [1, 3]), ('b', [2])]))
  File "/build/third_party/py/apache_beam/pipeline.py", line 425, in __exit__
    self.run().wait_until_finish()
  File "/build/third_party/py/apache_beam/pipeline.py", line 405, in run
    self._options).run(False)
  File "/build/third_party/py/apache_beam/pipeline.py", line 418, in run
    return self.runner.run_pipeline(self, self._options)
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner.py", line 265, in run_pipeline
    default_environment=self._default_environment))
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner.py", line 268, in run_via_runner_api
    return self.run_stages(*self.create_stages(pipeline_proto))
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner.py", line 355, in run_stages
    safe_coders)
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner.py", line 449, in run_stage
    elements_by_window = _WindowGroupingBuffer(si, value_coder)
  File "/build/third_party/py/apache_beam/runners/portability/fn_api_runner.py", line 191, in __init__
    self._key_coder = coder.wrapped_value_coder.key_coder()
  File "/build/third_party/py/apache_beam/coders/coders.py", line 177, in key_coder
    raise ValueError('Not a KV coder: %s.' % self)
ValueError: Not a KV coder: BytesCoder.
```

Reply:
Several other tests this this file fail with similar error.

Reply:
This test passes on my machine. Which Python 3 version are you using?

Reply:
Could you please check if it passes on 3.6?

Reply:
My version was 3.6.7

Reply:
Since we are targeting Python 3.5 right now, we don't have to add 3.6 fixes in this PR, but I would prefer to unskip such tests only for the 3.x versions, that the test is passing with.

Reply:
Only unskipping tests for passing versions would mean we would need to check the tests for multiple versions every time, which would take some time. Leaving all tests unskipped for higher versions also doesn't seem like the best approach, since most will be fixed for all versions anyway.

I also think some other already unskipped tests will fail on Python 3.6+. For instance some typehints tests related to type inference, which uses byte code inspection, which was changed in Python 3.6.

Looking at the fastest way forward, I would like to focus solely on 3.5 for now, and add skip statements for 3.6 when a 3.6 test environment is added. What do you think?

Reply:
Ok, that's fine. I created a JIRA tracking this failure: https://issues.apache.org/jira/browse/BEAM-6429.